plugins/backbone.js

```diff
@@ -13,9 +13,20 @@ if (!Backbone) {
 
 function makeBackboneEventsOn(oldOn) {
   return function BackboneEventsOn(name, callback, context) {
-    var _callback = callback._callback || callback;
-    callback = Raven.wrap(callback);
-    callback._callback = _callback;
+    var wrapCallback = function (cb) {
+      var _callback = cb._callback || cb;
+      cb = Raven.wrap(cb);
+      cb._callback = _callback;
+      return cb;
+    };
+    if (typeof name === 'object') {
```

Comment:
Understood, and fixed. I was just copying how Backbone handles this check internally (https://github.com/jashkenas/backbone/blob/master/backbone.js#L223), but we might as well make it more robust here.